metals/src/main/scala/scala/meta/internal/metals/FileDecoderProvider.scala

```diff
@@ -67,34 +66,38 @@ final class FileDecoderProvider(
    * jar:file:///jarPath/jar-sources.jar!/packagedir/file.java
    *
    * Examples...
-   * metalsDecode:file:///somePath/someFile.java.javap-verbose?decoder=javap&verbose=true
-   * metalsDecode:file:///somePath/someFile.scala.javap-verbose?decoder=javap&verbose=true
-   * metalsDecode:file:///somePath/someFile.class.javap-verbose?decoder=javap&verbose=true
+   * javap:
+   * metalsDecode:file:///somePath/someFile.java.javap
+   * metalsDecode:file:///somePath/someFile.scala.javap
+   * metalsDecode:file:///somePath/someFile.class.javap
    *
-   * metalsDecode:file:///somePath/someFile.java.javap?decoder=java
-   * metalsDecode:file:///somePath/someFile.scala.javap?decoder=javap
-   * metalsDecode:file:///somePath/someFile.class.javap?decoder=javap
+   * metalsDecode:file:///somePath/someFile.java.javap-verbose
+   * metalsDecode:file:///somePath/someFile.scala.javap-verbose
+   * metalsDecode:file:///somePath/someFile.class.javap-verbose
    *
-   * metalsDecode:file:///somePath/someFile.java.semanticdb-compact?decoder=semanticdb&format=compact
-   * metalsDecode:file:///somePath/someFile.java.semanticdb-detailed?decoder=semanticdb
+   * semanticdb:
+   * metalsDecode:file:///somePath/someFile.java.semanticdb-compact
+   * metalsDecode:file:///somePath/someFile.java.semanticdb-detailed
    *
-   * metalsDecode:file:///somePath/someFile.scala.semanticdb-compact?decoder=semanticdb&format=compact
-   * metalsDecode:file:///somePath/someFile.scala.semanticdb-detailed?decoder=semanticdb
+   * metalsDecode:file:///somePath/someFile.scala.semanticdb-compact
+   * metalsDecode:file:///somePath/someFile.scala.semanticdb-detailed
    *
-   * metalsDecode:file:///somePath/someFile.java.semanticdb.semanticdb-compact?decoder=semanticdb&format=compact
-   * metalsDecode:file:///somePath/someFile.java.semanticdb.semanticdb-detailed?decoder=semanticdb
+   * metalsDecode:file:///somePath/someFile.java.semanticdb.semanticdb-compact
+   * metalsDecode:file:///somePath/someFile.java.semanticdb.semanticdb-detailed
    *
-   * metalsDecode:file:///somePath/someFile.scala.semanticdb.semanticdb-compact?decoder=semanticdb&format=compact
-   * metalsDecode:file:///somePath/someFile.scala.semanticdb.semanticdb-detailed?decoder=semanticdb
+   * metalsDecode:file:///somePath/someFile.scala.semanticdb.semanticdb-compact
+   * metalsDecode:file:///somePath/someFile.scala.semanticdb.semanticdb-detailed
    *
-   * metalsDecode:file:///somePath/someFile.scala.tasty-detailed?decoder=tasty
-   * metalsDecode:file:///somePath/someFile.tasty.tasty-detailed?decoder=tasty
+   * tasty:
+   * metalsDecode:file:///somePath/someFile.scala.tasty-detailed
+   * metalsDecode:file:///somePath/someFile.tasty.tasty-detailed
```

Comment:
One other things, we have all the details of how these uri's should be created in here, but can we either move this to where the command is defined or copy it there as well? Mainly just so that it appears [here](https://scalameta.org/metals/docs/integrations/new-editor/#decode-file) because right now it's impossible for clients to know how to integrate this without looking into the code.

Reply:
Also, we have `tasty-detailed`, but there isn't a non detailed version right? Do we need to have the `-detailed` part?

Reply:
Also other small nit, for `javap` if we want compact we just send in `.javap`, whereas for semanticdb we send in `.semanticdb.compact`. What do you think about aligning all of these so from the outside, they are all the same meaning that instead of having to ensure for javap you use `verbose`, but for `semanticdb` you use `detailed` you'd just use the same value. So then no matter the decoder you'd choose `compact` or `detailed`? (or `verbose` if you like that better).

Reply:
Yes, there is only one way to decode `.tasty`, so the additional extension isn't necessary and can be omitted.
About aligning the rest of the commands, extensions come from options available in `javap` and `metap` commands so both keeping them as they are now and aligning them makes sense to me. I'll just leave it up to you :D

Reply:
Ok, I decided not to change extensions for javap and semanticdb. Now they nicely correspond to the options in these commands.
Also, although there is only one way to decode tasty file it's good to have an additional extension. Maybe `tasty-decoded` will be better than `tasty-detailed`?

Reply:
Yea I think either is probably fine.